app/assets/javascripts/discourse/app/components/post/menu.gjs

```diff
@@ -395,6 +396,11 @@ export default class PostMenu extends Component {
     return items;
   }
 
+  @action
+  expandIfOnlyOneHiddenButton() {
+    this.collapsed = this.renderableCollapsedButtons.length > 1;
```

Comment:
Just wondering but can you do this on the `@tracked` property directly instead of in `didInsert`?

Reply:
That was my first approach. The problem is that `collapsed` is part of logic to find all buttons and it caused infinite loop

https://github.com/discourse/discourse/blob/main/app/assets/javascripts/discourse/app/components/post/menu.gjs#L369